app/api/private-cloud/products/[licencePlate]/comments/route.ts

```diff
@@ -1,14 +1,20 @@
 import { z } from 'zod';
 import createApiHandler from '@/core/api-handler';
-import { CreatedResponse, OkResponse } from '@/core/responses';
+import { CreatedResponse, OkResponse, BadRequestResponse } from '@/core/responses';
 import { PermissionsEnum } from '@/types/permissions';
 import { createOp } from './_operations/create';
 import { listOp } from './_operations/list';
 
-const CreateCommentBodySchema = z.object({
-  text: z.string().min(1, 'The comment text must not be empty'),
-  projectId: z.string(),
-});
+const CreateCommentBodySchema = z
```

Comment:
1. the schema should be `camelCase`.
2. let's make `productId` required and `requestId` optional.